IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -273,6 +273,7 @@ - (BOOL)matchesWithHomeAccountId:(nullable NSString *)homeAccountId
     if (homeAccountId && 
         ![self.homeAccountId.msidNormalizedString isEqualToString:homeAccountId.msidNormalizedString])
     {
+        MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, nil, @"(MSIDCredentialCacheItem) cached item did not have a valid home account Id. Actual: %@, expected: %@",self.homeAccountId.msidNormalizedString,homeAccountId.msidNormalizedString);
```

Comment:
Home account id is EUPI, please wrap it in PII macros

Reply:
I see. thanks for highlighting it.  Perhaps it would be useful for us to compile a list of attributes that require EUPI (unless there is one already somewhere) and share in the knowledge base as a guideline for our team.